dojo/unittests/test_trivy_parser.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase
+from dojo.tools.trivy.parser import TrivyParser
+from dojo.models import Test
+
+import os.path
+
+
+def sample_path(file_name):
+    return os.path.join('dojo/unittests/scans/trivy', file_name)
+
+
+class TestTrivyParser(TestCase):
+
+    def setUp(self):
+        self.dojo_test = Test()
+
+    def test_mixed_scan(self):
+        with open(sample_path('trivy_mix.json')) as test_file:
+            trivy_parser = TrivyParser(test_file, self.dojo_test)
+        self.assertEqual(len(trivy_parser.items), 6)
+
+    def test_python_scan(self):
+        with open(sample_path('trivy_v0.2.1_python3.json')) as test_file:
+            trivy_parser = TrivyParser(test_file, self.dojo_test)
+        self.assertEqual(len(trivy_parser.items), 1427)
```

Comment:
Quite a few findings here. Maybe reduce it to a couple dozens to keep the test meaningful but eventually a bit faster?

Reply:
I say remove it in favor of additional tests for handling zero and one findings. Since this file is already submitted to the sample-scan repo, and will be merged after this, it is redundant to have it here as well.

Reply:
Agreed. Removed the huge trivy JSON report together with the corresponding unit test.